src/commands/stop.js

```diff
@@ -3,6 +3,7 @@ const sm = require("../index");
 
 sm.command(["stop", "disconnect", "leave"], (msg) => {
     let cmdGuild = sm.data[msg.guild.id];
+    sm.data[msg.guild.id].loop = false;
```

Comment:
This isn't required as it is deleted in the disconnect listener to reset it for the next join event.
https://github.com/zacimac/SimpleMusic/blob/f7c7d6ea70286439ccaa793c04d7dcba96b1bdf4/src/commands/events.js#L40